examples/step-63/step-63.cc

```diff
@@ -390,14 +425,19 @@ namespace Step63
       values[i] = BoundaryValues<dim>::value(points[i], component);
   }
 
+
+
+  // @sect3{Streamline Diffusion}
+
+  // Streamline diffusion stabilization term. Value defined in
+  // 'On discontinuity–capturing methods for convection–diffusion
+  // equations' (cite?)
   template <int dim>
```

Comment:
Not sure where to talk about streamline diffusion. Here I'm assuming that I will talk about it in the Introduction.